CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.3
+
```

Comment:
nit: all the other lines don't have this space.

Reply:
Okay ... I'm glad that we both seem to have no real problems 🤣